tools/report.sh

```diff
@@ -8,9 +8,13 @@ fi
 { # this ensures the entire script is downloaded #
 NAMESPACE=""
 CLUSTER=""
+BRIDGE=""
+CONNECT=""
+MM2=""
 KUBECTL_INSTALLED=false
 OC_INSTALLED=false
 KUBE_CLIENT="kubectl"
+OUT_DIR="$(mktemp -d)"
```

Comment:
Should this be called only when `--out-dir` was not specified?

Reply:
This is a default value, which is overwritten when you specify `--out-dir`.

Reply:
Right. But you call `mktemp -d` then even if you end up not needing it.

Reply:
Agree. We want to avoid creating folders when they are not used. Let me fix that.